docs/source/getting_started/getting_started.md

```diff
@@ -216,11 +217,98 @@ Alternatively a history of interactions can be used instead of a user id to ask
 }
 ```
 
+```{note}
 Please note: `"exclude_seen": true` (default true) filters out documents from search results, that were interacted with by the user or have been provided in the history.
+```
+
+# Filters
+
+Finding specific documents in large datasets based on a key-phrase or their relation to other documents can often be challenging. To address this issue, we can employ a structured filter on one or more of the `properties` fields to narrow down the search scope.
+
+The `filter` functionality is available in the [`/semantic_search`](https://docs.xayn.com/front_office.html#tag/search/operation/getSimilarDocuments) and [`/users/{user_id}/personalized_documents`](https://docs.xayn.com/front_office.html#tag/search/operation/getPersonalizedDocuments) endpoints, and it involves a two-step process:
+
+1. Indexing the desired property field for the filter to operate on.
+2. Applying the `filter` in the request to `/semantic_search` or `/users/{user_id}/personalized_documents`.
+
+```{note}
+Please note that the __first step__ is necessary to leverage the filtering at all.
+```
+
+## Indexing a filter property
+
+First lets check which properties are already indexed: 
+
+```bash
+curl --location 'https://<url>/documents/_indexed_properties' \
+--header 'authorizationToken: <back_office_token>>'
+```
+
+This returns just the `publication_date`, which is indexed by default (__Deprecated__).
```

Comment:
:art: the indexing itself is not deprecated, but the usage of the `published_after` request option is deprecated. the user can still filter by that via the `filter` request option, for example like
```json
"filter": { "publication_date": { "$gte": "2000-01-01T00:00:00Z" } }
```
i'm not sure if we are also going to deprecate the default indexing of `publication_date` in the future, maybe if we see that the tenants start to use their own name for indexing date fields.